composer.json

```diff
@@ -17,8 +17,8 @@
         "guzzlehttp/guzzle": "^6.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.7",
-        "arbor-education/php-coding-standard": "@dev",
+        "phpunit/phpunit": "^4.7 || ^9.0",
+        "arbor-education/php-coding-standard": "dev-IT-10201"
```

Comment:
Is this some temp solution? @nemke

Reply:
Yep, that is why it was under `do-not-merge` tag